packages/swarm-docs/src/components/textInputExample.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { TextInput } from '@meetup/swarm-components';
+
+const Example = () => (
```

Comment:
Can we update the example to allow for users to update the text?

Reply:
done